### ISSUE TITLE
Getting this issue when using workflow from repository

### ISSUE THREAD

**Issue by RYG81** [MrForExample/ComfyUI-3D-Pack#99]
While tying to run this workflow -
https://github.com/MrForExample/ComfyUI-3D-Pack/blob/main/_Example_Workflows/CRM_Img_to_Multi-views_to_CCMs_to_Mesh.json

getting below error

Error occurred when executing [Comfy3D] CRM Images MVDiffusion Model:

Sizes of tensors must match except in dimension 2. Expected size 630 but got size 64 for tensor number 1 in the list.

File "D:\000AI\ComfyUI\ComfyUI\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\000AI\ComfyUI\ComfyUI\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\000AI\ComfyUI\ComfyUI\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\000AI\ComfyUI\ComfyUI\custom_nodes\ComfyUI-3D-Pack\nodes.py", line 1780, in run_model
pixel_img = torch_img_to_pil_rgba(reference_image, reference_mask)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\000AI\ComfyUI\ComfyUI\custom_nodes\ComfyUI-3D-Pack\shared_utils\image_utils.py", line 24, in torch_img_to_pil_rgba
single_image = torch.cat((img, mask), dim=2).detach().cpu().numpy()
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by MrForExample** [comment]
Hi, seems the load image node doesn't recognize the mask from your image, you can try to upscale your image a bit, and make sure your image has bit depth higher than 24. Cheers 👍

**Comment by RYG81** [comment]
Thanks
